src/vsr/replica.zig

```diff
@@ -9063,13 +9216,13 @@ fn message_body_as_view_headers(message: *const Message) vsr.Headers.ViewChangeS
 
 /// Asserts that the headers are in descending op order.
 /// The headers may contain gaps and/or breaks.
-fn message_body_as_headers(message: *const Message) []const Header {
+fn message_body_as_prepare_headers(message: *const Message) []const Header.Prepare {
```

Comment:
In the spirit of pushing complexity to the definiton site, might take in an `anytype` here and comptime assert that the type is right, primarily to get rid of .base() calls at the call-site. Not sure it is worth it though.

Reply:
I lean towards leaving these functions as-is -- they aren't used as often as (e.g.) `unref()`.
`anytype` cleans up the call site, but at the cost of worse type signatures (for both readability and type errors).